.gitignore

```diff
@@ -2,3 +2,4 @@
 node_modules/
 *.log
 /stats.json
+.idea/
```

Comment:
I think you should use your own global git exclusion config - as you can see .gitignore here doesn't list any IDE artifacts.